Dart/thirdPartySrc/analysisServer/com/google/dart/server/LSPDartTextDocumentContentConsumer.java

```diff
@@ -0,0 +1,16 @@
+// Copyright 2000-2024 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
```

Comment:
I don't know if copyrights are still needed in each file. But if so, I guess it should be the same as in neighbor files (Copyright (c) 2024, the Dart project authors...)

Reply:
Done.